src/pages/register.tsx

```diff
@@ -0,0 +1,211 @@
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Register = () => {
+    const inputStyle =
+        'w-full bg-input-gray border-input-gray rounded-full p-2 px-3 mb-4';
+    return (
+        <div className="flex lg:justify-end">
+            <form className=" bg-white w-full lg:w-2/3 mt-40 lg:mt-0 rounded-t-[20px] lg:rounded-none lg:rounded-l-[20px] flex flex-col justify-start items-center text-purple font-ibm">
+                <h1 className="text-3xl font-bold mt-12 mb-2">ลงทะเบียน</h1>
+                <div className="flex flex-col justify-start items-center w-10/12 pt-6 lg:flex-row-reverse lg:justify-between">
+                    <div className="w-1/2 mb-6 lg:w-1/4">
+                        <div className="flex flex-row justify-between items-center">
+                            <div>อัปโหลดรูป</div>
+                            <button className="bg-[#E36130] flex flex-row justify-center items-center py-1 px-3 rounded-full">
+                                <Image
+                                    src="/images/heroicons-solid/pencil-square.svg"
+                                    alt=""
+                                    height={16}
+                                    width={16}
+                                />
+                                <p className="text-white text-xs">แก้ไข</p>
+                            </button>
+                        </div>
+                        <div className="w-full h-52 relative rounded-2xl overflow-hidden  my-4">
+                            <Image
+                                src="/images/pfp-placeholder.png"
```

Comment:
For a local image, I believe it would be better to import it first. Here is an example code:
```ts
...
import ProfilePlaceholder from '@/public/images/pfp-placeholder.png'
...
  <Image
    src={ProfilePlaceholder}
    ...
  />
...
```